ee/server/service/mdm_profiles.go

```diff
@@ -33,6 +33,8 @@ var fileVaultProfileTemplate = template.Must(template.New("").Option("missingkey
 			<false/>
 			<key>DeferForceAtUserLoginMaxBypassAttempts</key>
 			<integer>1</integer>
+			<key>ForceEnableInSetupAssistant</key>
+			<true/>
```

Comment:
did the issue specify what to do with existing FileVault profiles? what about profiles that are already delivered for a team?

Reply:
The comments in the issue talk about several related but IMO different feature requests. I took what was in the description to only be about enforcing FileVault during the ADE flow, but if I missed the mark definitely lmk!

Reply:
when FileVault is enabled, we create a profile and save it in `mdm_apple_configuration_profiles`

https://github.com/fleetdm/fleet/blob/2dd3f2cd1fec1553cc885d133565e96879d15dce/ee/server/service/mdm.go#L191-L214

profiles stored there won't have this setting. I think we need to address this case in a migration or request for the IT admin to enable/disable FileVault

Reply:
this is how we did it in the past https://github.com/fleetdm/fleet/blob/13157-fv-escrow/server/datastore/mysql/migrations/tables/20230711144622_SetFileVaultMaxBypassAttempts.go

Reply:
> when FileVault is enabled, we create a profile and save it in mdm_apple_configuration_profiles

Right! 

> I think we need to address this case in a migration or request for the IT admin to enable/disable FileVault

Ah gotcha, I can work on a migration (so that it's seamless)

Reply:
> this is how we did it in the past
Nice! thanks for the context!

Reply:
> what about profiles that are already delivered for a team?

Is there anything to do in this case? If a host is already enrolled, the enforcement at setup time won't matter, and the next time that machine is enrolled (e.g. it's wiped and sent to some new employee) it'll get the new profile.

Reply:
I think you're right and that case is fine, we just need to make sure to include the migration

Reply:
having that said, since we're going to change the profile, it'll get re-delivered to all hosts anyways